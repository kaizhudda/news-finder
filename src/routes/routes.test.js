import React from 'react';
import { shallow } from 'enzyme';
import Routes from './index';

let wrapped = shallow(<Routes />);

describe('Routes', () => {
  it('should render the Routes Component', () => {
    expect(wrapped).toMatchSnapshot();
  });

  it('should have 2 route', () => {
    expect(wrapped.find('Route').length).toBe(2);
  });
});