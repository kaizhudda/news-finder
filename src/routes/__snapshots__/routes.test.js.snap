// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes should render the Routes Component 1`] = `
<Switch>
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(NewsResults)",
        "type": [Function],
      }
    }
    exact={true}
    path="/"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(Author)",
        "type": [Function],
      }
    }
    exact={true}
    path="/author/:username"
  />
  <Redirect
    to="/"
  />
</Switch>
`;
